website/docs/cdktf/test/unit-tests.mdx

```diff
@@ -5,24 +5,32 @@ description: Write assertions and snapshot tests for your CDK for Terraform appl
 
 # Unit Tests
 
-Testing your application can give you faster feedback cycles and guard you against unwanted changes. Currently, testing is only supported in Typescript with jest.
+Testing your application can give you faster feedback cycles and guard you against unwanted changes. Testing is currently supported in Typescript with jest and compatible with any testing framework that supports assertions for all other languages.
 
-We generate all files necessary to run jest when you run `cdktf init` so that you can start writing tests right away. If you want to add jest to an existing project, please [follow their guide](https://jestjs.io/docs/getting-started). Then, you need to add these lines in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array):
+For Typescript we generate all files necessary to run jest when you run `cdktf init` so that you can start writing tests right away. If you want to add jest to an existing Typescript project, please [follow their guide](https://jestjs.io/docs/getting-started). Then, you need to add these lines in a [setup file](https://jestjs.io/docs/configuration#setupfiles-array):
```

Comment:
Could you do a quick follow up and add the testing setup to all of the templates as well? Should be as simple as copying the example tests you already wrote into a file (commented out) and adding the testing helper and testing instructions